### ISSUE TITLE
Broken examples using Panel (& Fix)

### ISSUE THREAD

**Issue by FlorianKrach** [pyscript/examples#17]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

## Problem

All the PyScript examples using Panel provided at https://pyscript.com/@examples currently don't work, see for example https://pyscript.com/@examples/streaming-in-panel/latest. I used "Safari" and "Chrome", but the problem seems to be independent of the browser.

As far as I can judge (but I am not an expert), the issue seems to be that there was some update in Bokeh or Panel during the last weeks, which now need to install two libraries for which no pure python wheels are available (`ValueError: Can't find a pure Python 3 wheel for: 'tornado>=6.2', 'contourpy>=1.2'`).
As a result, this breaks all PyScript apps using Panel, where Bokeh and Panel are installed via
```
packages = ["bokeh==3.2.2", "panel"]
```
or similarly.

## Fix

A way to circumvent this is by installing the Panel and Bokeh packages directly from the wheels provided by Holoviz, as was outlined here: https://github.com/holoviz/panel/issues/4572#issuecomment-1490749304. In particular, by replacing the above by:
```
packages=[
    "https://cdn.holoviz.org/panel/0.14.4/dist/wheels/bokeh-2.4.3-py3-none-any.whl",
    "https://cdn.holoviz.org/panel/0.14.4/dist/wheels/panel-0.14.4-py3-none-any.whl",
]
```
Note: there are already newer versions of Bokeh and Panel available, but this also works (at least when adjusting the script tags accordingly).

Since the provided PyScript examples currently don't work, I thought you might want to update them using this fix. I am sorry in case I reported the problem at the wrong place. Thank you for the great tool you provide with PyScript!

### Console info

```shell
Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python3.11/site-packages/micropip/_commands/install.py", line 146, in install
    raise ValueError(
ValueError: Can't find a pure Python 3 wheel for: 'contourpy>=1.2', 'tornado>=6.2'
See: https://pyodide.org/en/stable/usage/faq.html#why-can-t-micropip-find-a-pure-python-wheel-for-a-package


    at new_error (pyodide.asm.js:9:12519)
    at pyodide.asm.wasm:0x158827
    at pyodide.asm.wasm:0x15fcd5
    at _PyCFunctionWithKeywords_TrampolineCall (pyodide.asm.js:9:123052)
    at pyodide.asm.wasm:0x1a3091
    at pyodide.asm.wasm:0x289e4d
    at pyodide.asm.wasm:0x1e3f77
    at pyodide.asm.wasm:0x1a3579
    at pyodide.asm.wasm:0x1a383a
    at pyodide.asm.wasm:0x1a38dc
    at pyodide.asm.wasm:0x2685c5
    at pyodide.asm.wasm:0x26e3d0
    at pyodide.asm.wasm:0x1a3a04
    at pyodide.asm.wasm:0x1a3694
    at pyodide.asm.wasm:0x15f45e
    at Module.callPyObjectKwargs (pyodide.asm.js:9:81732)
    at Module.callPyObject (pyodide.asm.js:9:82066)
    at wrapper (pyodide.asm.js:9:58562)
```

**Comment by WebReflection** [comment]
this is a duplicate of https://github.com/pyscript/examples/issues/16 although this has better details and workarounds, thank you!

I am keeping this open and I am going to close the other.

**Comment by ambv** [comment]
I also stumbled upon this issue, in my case I worked around it by simply pinning panel to `panel==1.3.8`. It was my intuition that a version from the time when the example was last updated is probably going to work 🤓

I have two general suggestions:

1. In user-visible examples maybe we could display the exception text instead of the "Loading" screen forever? Loading up the Console is kind of an afterthought and not the best experience.

2. Maybe you want to pin all dependencies in examples to avoid them stopping working because of updates?
